TEST_APPS/readme.md

```diff
@@ -0,0 +1,115 @@
+## Running IceTea tests located under mbed-os
+
+### Structure
+
+mbed-os has a folder called TEST_APPS that contains everything related to IceTea -testing.
+There are currently 3 folders
+
+- device - contains all the different test applications you can flash to your board
+- icetea-plugins - contains plugins that are being used by some of the testcases, needed for the test execution
+- testcases - contains IceTea testcases written in Python
+
+The testcases and test applications have a dependency
+
+### Preparing your work environment
+
+#### Prerequisities
+
+You should have IceTea and forked mbed-cli that supports IceTea, installed.
+
+#### Selecting the network interface to use
+
+Depending on a device, there might be a default network interface type defined in the mbed-os/targets/targets.json, which is used to locate a test-config file by default. 
+If there is not, or you want to use a different interface than the default, you need to provide a relevant test-config -file to the mbed test with --test-config option.
+The test-config file contains the necessary information for the test application, there are some test-config files located under mbed-os/tools/test-configs.
+
+### Running the tests
+
+Now that the interface has been selected you can run the icetea tests from the mbed-os root on your command line by
+
+`>mbed test -m <target> -t <toolchain> --icetea`
```

Comment:
I asked in the unittest PR, just to be sure this is fine also here (commands/arguments alignment for cli) - how we can test (unit/icetea/app tests) - using one command `mbed test` and `--argument`.

Reply:
This is handled in unit testing.. it add new command "mbed unittest" because it is so different than icetea or greentea